app/uk/gov/hmrc/agentsubscriptionfrontend/models/Address.scala

```diff
@@ -137,7 +136,4 @@ object Address{
 
     OFormat[Address](reads, formatAddressValue)
   }
-
-  val renderErrors: PostCodeError => String = (errors: PostCodeError) => errors.foldLeft("")(_+", "+_).substring(1)
```

Comment:
I can write unit tests for this anon function. It has the same functionality with def